packages/pdf-viewer/main.tsx

```diff
@@ -20,7 +20,10 @@ document.documentElement.setAttribute('data-theme', appearance);
 
 function App() {
 	if (type === 'mini') {
-		return <MiniViewerApp pdfPath={url} isDarkTheme={appearance === 'dark'} anchorPage={anchorPage ? Number(anchorPage) : null} pdfId={pdfId} />;
+		return <MiniViewerApp pdfPath={url}
+			isDarkTheme={appearance === 'dark'}
+			anchorPage={anchorPage ? Number(anchorPage) : null}
```

Comment:
Convert anchorPage to a number (or null) at the point where you get the attribute (so above). Otherwise you have to cast it to int everywhere.